### PR TITLE
Edits for 2.2

### PR DESCRIPTION
**Pull request by mansam**

### CODE REVIEW

File: documentation/modules/about-cold-warm-migration.adoc [hunk 1]
```diff
@@ -29,10 +29,10 @@ The VM disks are copied incrementally using link:https://kb.vmware.com/s/article
 ====
 You must enable CBT on the source VMs and the VM disks.
 
-A VM can support up to 28 CBT snapshots. If that limit is exceeded, a `warm import retry limit reached` error message is displayed. If the VM has preexisting CBT snapshots, it will reach this limit sooner.
+A VM can support up to 28 CBT snapshots. If the source VM already has too many CBT snapshots and the Migration Controller is not able to create a new one, warm migration may fail. The Migration Controller will clean up each snapshot it creates after it is finished with it.
 ====
 
-The precopy stage runs until either the cutover stage starts or the maximum number of CBT snapshots is reached.
+The precopy stage runs indefinitely until the cutover stage is reached.
```
Comment (apinnick) [inline documentation/modules/about-cold-warm-migration.adoc:35]: Should this be "until the cutover stage is triggered manually or the **scheduled time** of the cutover stage is reached"?

File: documentation/modules/virt-migration-workflow.adoc [hunk 2]
```diff
@@ -18,14 +18,12 @@ You can use the detailed migration workflow to troubleshoot a failed migration.
 
 The workflow describes the following steps:
 
-. When you run a migration plan, the Migration Controller creates a `VirtualMachineImport` custom resource (CR) for each source virtual machine (VM).
-. The Virtual Machine Import Controller validates the `VirtualMachineImport` CR and generates a `VirtualMachine` CR.
-. The Virtual Machine Import Controller retrieves the VM configuration, including network, storage, and metadata, linked in the `VirtualMachineImport` CR.  
+. When you run a migration plan, the Migration Controller creates a VirtualMachine custom resource (CR) for each source virtual machine (VM), and a DataVolume CR for each of the source VM's disks.
```
Comment (apinnick) [inline documentation/modules/virt-migration-workflow.adoc:21]: VirtualMachine and DataVolume need to be marked up with backticks.

File: documentation/modules/about-cold-warm-migration.adoc (hunk 1 above)
Comment (mansam) [reply documentation/modules/about-cold-warm-migration.adoc:35]: Yes, that would be more accurate.

**Comment by apinnick** [comment]
Changes copied to https://github.com/konveyor/forklift-documentation/pull/281.

**Pull request closed without merging**